CHANGELOG.md

```diff
@@ -7,9 +7,12 @@
 - **BREAKING**: For `TimeoutError`, change the error message to include the
   request's url. Was: `Request timed out`, 
   now: `Request to "example.com" timed out.`
+- **BREAKING**: Remove default exports.
 
 ### Added
 - Add a `.requestUrl` property to all errors, for easier logging/diagnostics.
+- Add `PUT` to allowed proxied methods.
+- Fix `.extend()` to return a proxy (with default headers, etc).
```

Comment:
Please move this under `### Fixed` to keep with our style.

Reply:
Changed, thanks.